examples/amp-story/amp-story-shopping.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html amp lang="en">
+<html amp lang="en" dir="rtl">
```

Comment:
Please revert the rtl changes in the example doc before checking it in. 

Adding a visual test with `dir="rtl"` could be good if we want to keep an example and prevent regressions of this work.

Reply:
Sounds good, I have added the visual diff test amp-story-shopping-rtl.html which contains the dir-"rtl" attribute.